k8s/demo/common/probate/probate-back-office.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     java:
```

Comment:
this file should probably be removed otherwise I think it will fail for a duplicate resource and break flux

Reply:
Yeah I know, having to add it to remove it to resolve conflicts. hoping not to have to scrap this PR and start all over..